enums.go

```diff
@@ -792,3 +792,39 @@ const (
 	// Filestore PDF mime type
 	TILEDB_MIME_PDF = FileStoreMimeType(C.TILEDB_MIME_PDF)
 )
+
+// DataOrder
+type DataOrder int8
+
+const (
+	// Unordered dimension label.
+	TILEDB_UNORDERED_DATA DataOrder = C.TILEDB_UNORDERED_DATA
+	// Ordered dimension label with increasing values.
+	TILEDB_INCREASING_DATA DataOrder = C.TILEDB_INCREASING_DATA
+	// Ordered dimension label with decreasing values.
+	TILEDB_DECREASING_DATA DataOrder = C.TILEDB_DECREASING_DATA
+)
+
+// String Returns string representation of a DataOrder
+func (d DataOrder) String() (string, error) {
+	var dataOrderStr *C.char
+	ret := C.tiledb_data_order_to_str(C.tiledb_data_order_t(d), &dataOrderStr)
+	if ret != C.TILEDB_OK {
+		return "", fmt.Errorf("Error converting DataOrder to string: %d", d)
+	}
+
+	return C.GoString(dataOrderStr), nil
+}
+
+// FromString Converts from a string to the equivalent DataOrder enum
+func (d *DataOrder) FromString(name string) error {
```

Comment:
This seems like an odd API:

```go
var myDO DataOrder
myDO.FromString("someStr")
```

I would expect something like:

```go
order, err := tiledb.DataOrderFromString(someStr)
```

Reply:
Correct. Made it a package method